transport_plugins/bled112/iotile_transport_bled112/async_packet.py

```diff
@@ -50,9 +50,41 @@ def read_packet(self, timeout=3.0):
         except Empty:
             raise InternalTimeoutError("Timeout waiting for packet in AsyncPacketBuffer")
 
+#80 2a 06 00 ca 00 00 90 29 81
+#25 cb 01 ff 1f 02 01 06 1b 16
+#dd fd 00 90 00 00 00 00 00 00 
+#23 16 00 00 a0 01 40 10 00 00
+#00 00 00 00 00 00  rssi: -54, type: 0, sender: 0090298125cb
+def packet_is_broadcast_v2(packet):
+    #Broadcast packets consist of 32 bytes for data, 10 for BLE packet header and 4 for bled112 bgapi header
+    if len(packet) != 46:
+        return False
+    #This identifies the bgapi packet as an event
+    if not (packet[0] == 0x80 and packet[2] == 6 and packet[3] == 0):
+        return False
+    #This identifies the event as a broadcast v2 packet
+    if not (packet[18] == 0x1b and packet[19] == 0x16 and packet[20] == 0xdd and packet[21] == 0xfd):
+        return False
+    return True
+
+def packet_is_dupe(packet, dedupe_dict):
+    mac = tuple(packet[6:11])
+
+    #Drop the sequence value from the packet, since that changes for each packet
+    data = packet[22:]
+
+    #print(f"Parsing from {mac}: {data}")
+    if dedupe_dict.get(mac, None) == data:
```

Comment:
dicts that have no key on `get` return `None` by default, no need to specify it as an alternate outcome.